src/TypeScriptToKotlinWalker.kt

```diff
@@ -43,9 +43,13 @@ private val COMPARE_BY_NAME = { a: Named, b: Named -> a.name == b.name }
 private val IS_NATIVE_ANNOTATION = { a: Annotation -> a.name == NATIVE }
 
 enum class ImplementationType(private val string: String) {
-    UNSPECIFIED(""), NO_IMPL(ts2kt.kotlin.ast.EQ_NO_IMPL), NO_IMPL_GETTER(PROPERTY_GETTER + EQ_NO_IMPL);
+    UNSPECIFIED(""), NO_IMPL(ts2kt.kotlin.ast.EQ_NO_IMPL), NO_IMPL_GETTER(PROPERTY_GETTER + EQ_NO_IMPL) {
```

Comment:
let's 
* rename it to NO_IMPL_PROPERTY and reformat
* replace`ts2kt.kotlin.ast.EQ_NO_IMPL` with `EQ_NO_IMPL`